src/sv-pipeline/scripts/identify_duplicates.py

```diff
@@ -44,11 +44,11 @@ def process_duplicates(vcf, fout):
                 record.stop,
                 record.info.get('SVTYPE'),
                 record.info.get('SVLEN'),
-                record.info.get('CHR2'),
-                record.info.get('END2'),
-                record.info.get('STRANDS'),
-                record.info.get('CPX_TYPE'),
-                record.info.get('CPX_INTERVALS')
+                record.info.get('CHR2') or "",
```

Comment:
Thanks for the fix. Can you give me access to the testing workspaces? 

I would use the built-in `.get(key, default)` syntax for consistency across our scripts
```suggestion
                record.info.get('CHR2', ""),
```

Reply:
Just shared the workspace with you, apologies for this oversight.

And thanks for the suggestion, just modified it to use this. Given the simplicity of the change, I did a quick sanity test locally, but let me know if I should re-build the docker / re-test with the `.get(key, default)` syntax as well.